test/electricity/force_between_two_electrically_charged_particles_at_rest_test.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    convert_to,
+    Quantity,
+    SI,
+    prefixes
+)
+from symplyphysics.laws.electricity import force_between_two_electrically_charged_particles_at_rest as couloumb_law_scalar
+
+# Description. With help of online calculator at https://www.wolframalpha.com/widgets/gallery/view.jsp?id=5227fa9a19dce7ba113f50a405dcaf09
+## The force between the charges q1 = 10 [nC] and q2 = -50 [nC], which are separated by a distance of r = 2 [m], is equivalent to -0.011234 [N].
+
+@fixture(name="test_args")
+def test_args_fixture():
+    q1 = Quantity(10 * prefixes.nano * units.coulomb)
+    q2 = Quantity(-50 * prefixes.nano * units.coulomb)
+    r = Quantity(0.02 * units.meter)
```

Comment:
In description distance is 2 meters. Check your calculations or update description.